common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/WebViewAuthorizationFragment.java

```diff
@@ -198,6 +198,10 @@ private void setUpWebView(@NonNull final View view,
         mWebView.getSettings().setJavaScriptEnabled(true);
         mWebView.requestFocus(View.FOCUS_DOWN);
 
+        // clear cookies from web view session before processing new login request
+        WebViewUtil.removeCookiesFromWebView(getContext());
+        WebViewUtil.removeSessionCookiesFromWebView(getContext());
```

Comment:
The documentation for both of these methods state that they [cannot be called from the UI thread](https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/dev/common/src/main/java/com/microsoft/identity/common/internal/ui/webview/WebViewUtil.java#L87)

Reply:
(In general though, I do not think we should take this PR -- see my other comment)

Reply:
+1 - I also don't know why are making this change. I think we want the cookies to be there. @ayushmeghwani can you explain the rationale behind this change?